docs/scenes/material-browser.html

```diff
@@ -678,10 +690,6 @@
 						guiMaterial( gui, mesh, material, geometry );
 						guiMeshPhysicalMaterial( gui, mesh, material, geometry );
 
-						// only use scene environment
-
-						light1.visible = false;
-						light2.visible = false;
```

Comment:
Why did you remove these two lines?

Reply:
Sheen effect and perhaps some others only really have an effect when directional lights are in the scene.  I found that leaving those other lights in helps make the impact of some of those effects more prominent.

Adding all three lights made things seem a bit washed out, so I only left the first two visible.

Reply:
The sheen example also works with just environmental lighting: https://threejs.org/examples/webgl_loader_gltf_sheen

Please undo both lines since the materials are often used with just `Scene.environment`.